site/docs/latest/deployment/kubernetes.md

```diff
@@ -97,10 +97,14 @@ $ zk-shell localhost 2181
 
 ### Deploy Bookies
 
-Once ZooKeeper cluster is Running, you can then deploy the bookies.
+Once ZooKeeper cluster is Running, you can then deploy the bookies. You can deploy the bookies either using a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) or a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/).
```

Comment:
We could mention that StatefulSet is only practical when deploying in a cloud environment (or any K8S that has persistent volumes available) while DaemonSet is using the local disks